docs/aop/user/apps/payone.md

```diff
@@ -2,4 +2,119 @@
 title: Payone
 description: Description
 template: howto-guide-template
----
\ No newline at end of file
+---
+
+The [Payone](https://www.payone.com/DE-en) app lets your customers make payments with common payment methods such as credit card and PayPal.
+
+{% info_block infoBox "Info" %}
+
+We support only credit card and PayPal for Payone. If you want to use either or both of these payment methods, you must have them in your Payone contract. However, even if your contract with Payone includes other payment methods, you can not use them, as we do not support them yet.
+
+{% endinfo_block %}
+
+You can have multiple accounts with Payone. For example, you can have different Payone credentials per store. However, in the Back Office UI, there is just a global account, so you can not have separate accounts for separate stores.
+
+## Payment method modes
+
+For the *Payone Credit Card* payment method, we support the following modes:
+
+- *Preauthorization and Capture*: After a customer entered the credit card details during the checkout, the merchant preauthorizes or reserves the payable amount on the customer’s credit card. As soon as the items have shipped, this amount is captured. Capture kicks off the process of moving money from the customer’s credit card to the seller’s account. The preauthorization and capture mode is good for physical goods. It ensures that in case the ordered items are no more available or the customer cancels the order, the seller does not have to return the money and pay anything for the additional transaction.
```

Comment:
```suggestion
- *Preauthorization and Capture*: After a customer entered the credit card details during the checkout, the merchant preauthorizes or reserves the payable amount on the customer’s credit card. As soon as the items have shipped, this amount is captured. Capture kicks off the process of moving money from the customer’s credit card to the seller’s account. The preauthorization and capture mode is the method of choice for physical goods. It ensures that in case the ordered items are not available anymore or the customer cancels the order before it is shipped, the seller does not have to transfer the money back to the customer's account and thereby avoids a chargeback. These usually result in additional costs for the merchant due to the transaction for the chargeback.
```